### ISSUE TITLE
Two events (different topics) are generated for project creation

### ISSUE THREAD

**Issue by vikasrohit** [topcoder-platform/tc-project-service#473]
It should raise only one kafka bus event, right now we are raising two events:

1. project.action.create
2. connect.notification.project.created

We should identify the places where `connect.notification.project.created` is still being used and get rid of it by using the new event standard `project.action.create`.

**Comment by vikasrohit** [comment]
Related issue #472. We might not need to do anything for this issue when we close #472.

**Comment by maxceem** [comment]
> Related issue #472. We might not need to do anything for this issue when we close #472.

Right, this issue is a particular case of the bigger task we have: **Notification should be handled using our new standard Kafka events and old custom events should be removed** (this is task 9 in the list of after V5 migration tasks https://github.com/topcoder-platform/tc-project-service/issues/435).

This big task includes:
- #472
- https://github.com/topcoder-platform/tc-notifications/issues/195

So I'm removing this issue from the milestone and mark it as on hold.
